src/go/rpk/pkg/cli/container/start.go

```diff
@@ -477,3 +493,44 @@ func nodeAddr(port uint) string {
 		port,
 	)
 }
+
+// getContainerErr attempts to fetch the latest stderr output from the first
+// Redpanda node. It may reveal reasons for failing to start.
+func getContainerErr(state *common.NodeState, c common.Client) (string, error) {
+	ctx, _ := common.DefaultCtx()
+
+	json, err := c.ContainerInspect(ctx, state.ContainerID)
+	if err != nil {
+		return "", fmt.Errorf("could not inspect container: %v", err)
+	}
+
+	reader, err := c.ContainerLogs(
+		ctx,
+		state.ContainerID,
+		container.LogsOptions{
+			ShowStdout: false,
+			ShowStderr: true,
+			Since:      json.State.StartedAt,
+		},
+	)
+	if err != nil {
+		return "", fmt.Errorf("could not get container logs: %v", err)
+	}
+
+	// Docker logs over the wire are multiplexed using stdcopy package. To
+	// demux this stream we need to use stdcopy.StdCopy. See:
+	// https://github.com/moby/moby/issues/32794#issuecomment-297151440
+	bErr := bytes.NewBufferString("")
```

Comment:
Why NewBufferString with an empty string? This just creates an empty buffer, and can be `bytes.Buffer`